src/controller/python/OpCredsBinding.cpp

```diff
@@ -319,6 +322,68 @@ void pychip_OnCommissioningStatusUpdate(chip::PeerId peerId, chip::Controller::C
     return sTestCommissioner.OnCommissioningStatusUpdate(peerId, stageCompleted, err);
 }
 
+/**
+ * By using pychip_OpCreds_AllocateControllerForCustomCommissioningFlow, we will not using the commissioning flow from the
```

Comment:
Needs to re-phrase this somehow without "we". Something like `The call of pychip.OpCreds.AllocateControllerFoCustomCommissioningFlow allows allocating a device commissioner that will perform commissioning without using the auto-commissioning flow present in the Controller class`

Reply:
done